tests/test_separator.py

```diff
@@ -0,0 +1,14 @@
+"""
+ Pylibui test suite.
+
+"""
+
+from pylibui.controls import HorizontalSeparator
+from tests.utils import WindowTestCase
+
+
+class SeparatorTest(WindowTestCase):
```

Comment:
Since Separator doesn't have any attributes that we can check, I think example is sufficient. Maybe we can completely remove this file. What do you think @joaoventura?